src/Build.UnitTests/BackEnd/AssemblyTaskFactory_Tests.cs

```diff
@@ -659,7 +659,7 @@ public void VerifySameFactoryCanGenerateDifferentTaskInstances()
 #endif
                     false);
                 Assert.NotNull(createdTask);
-                Assert.IsType<TaskHostTask>(createdTask);
+                Assert.IsNotType<TaskHostTask>(createdTask);
```

Comment:
This one worries me -- it seems to have been passing with the previous assertion. What was the actual type?

Reply:
Never mind, fixed in a later commit.

Reply:
Right.